app/views/citizens/other_assets/_second_home_conditional_checkbox.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="govuk-checkboxes__item">
+  <input class="govuk-checkboxes__input"
+         id="second_home"
+         name="second_home_checkbox"
+         type="checkbox"
+         value="yes"
+         <%= @form.second_home_checkbox_status %>
+         data-aria-controls="conditional-second-home" />
+  <label class="govuk-label govuk-checkboxes__label" for="second_home_checkbox">
+    <%= t('citizens.other_assets.show.second_home') %>
+  </label>
+</div>
+
+<div class="govuk-checkboxes__conditional govuk-checkboxes__conditional--hidden" id="conditional-second-home">
+  <div class="govuk-form-group">
+    <label class="govuk-label" for="second_home_value">
+      <%= t('citizens.other_assets.show.estimated_value') %>
+    </label>
+    <input class="govuk-input govuk-!-width-one-third"
+           id="second_home_value"
+           name="second_home_value"
+           value="<%= value_with_currency_unit(@form.second_home_value) %>"
+           pattern="\A[0-9][0-9,]+(.[0-9]{0,2})?\z" />
+  </div>
+
+  <div class="govuk-form-group">
+    <label class="govuk-label" for="second_home_mortgage">
+      <%= t('citizens.other_assets.show.mortgage') %>
+    </label>
+    <input class="govuk-input govuk-!-width-one-third"
+           id="second_home_mortgage"
+           name="second_home_mortgage"
+           value="<%= value_with_currency_unit(@form.second_home_mortgage) %>"
+           pattern="\A[0-9][0-9,]+(.[0-9]{0,2})?\z" />
```

Comment:
I think the `+` needs to be `*` in the middle, as currently single figure values fail the pattern. For example `1.0`

Also as it is likely this will be used a lot, I'd suggest moving it to a help something like:
```
def currency_pattern
  '\A[0-9][0-9,]*(.[0-9]{0,2})?\z'
end
```
Then here the code becomes:
`pattern="<%= currency_pattern %>" />`